src/smt/theory_str_noodler/decision_procedure.h

```diff
@@ -22,6 +22,17 @@ namespace smt::noodler {
         UNDERAPPROX
     };
 
+    enum class TransformationType {
```

Comment:
Maybe add a comment explaining, what we consider “transformations” to be and that this is used only to handle the specific transformations in formulae, not for generic formulae handling.

Reply:
I will add something. Maybe we should name it differently too, do you have any ideas?

Reply:
Resolved in person. We will add a comment and keep the name. We will standardize what “transformation” means to us.

Reply:
I added some small comment

Reply:
It should be sufficient.